Blaise-Shyaka/My-Mentor
# [#170345230] Create signup endpoint and its test

# What does this PR do?

This PR adds the signup endpoint with its tests. However, this endpoint uses temporary data structures.

# Description of tasks to be completed

create /auth/signup endpoint

# How should this be manually tested?

- Clone My-Mentor repository
- Open the folder using your favorite editor
- Open the terminal
- In the root directory, type npm test
- Check whether all the 5 tests pass
